packages/elvis/src/utilities/grid.scss

```diff
@@ -1,183 +1,153 @@
+@mixin create-breakpoint-classes($size, $breakpoint: '') {
+  .col-#{$breakpoint}#{$size} {
+    width: calc(100% / 12 * $size);
+  }
 
-The MIT License (MIT)
+  .order-#{$breakpoint}#{$size} {
+    order: $size;
+  }
 
-Copyright (c) 2011-2018 Twitter, Inc.
-Copyright (c) 2011-2018 The Bootstrap Authors
+  .offset-#{$breakpoint}#{$size} {
+    margin-left: calc(100% / 12 * $size);
+  }
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
+  .row-cols-#{$breakpoint}#{$size} > * {
+    width: calc(100% / $size);
+  }
+}
 
-The above copyright notice and this permission notice shall be included in
-all copies or substantial portions of the Software.
+.e-grid {
+  box-sizing: border-box;
 
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-THE SOFTWARE.
+  .row {
+    --offset: 24px;
 
-*/
+    display: flex;
+    flex-wrap: wrap;
+    margin-inline: calc(var(--offset) * -1);
 
-@use 'breakpoints';
-@use 'bootstrap/scss/bootstrap-grid.scss' with (
-  // Defining variables for grid that will overwrite bootstraps variables
-  $grid-breakpoints: (xs: 0, sm: 320px, md: 768px, lg: 1024px, xl: 1440px),
-  $grid-columns: 12,
-  $grid-gutter-width: 48px
-);
-
-// Making custom margin classes
-.e-grid {
-  box-sizing: border-box;
+    &.e-grid-gutters-ext.e-grid-gutters-vertical [class^='col'],
+    &.e-grid-gutters-int.e-grid-gutters-vertical [class^='col'] {
+      padding-block: var(--offset) !important;
+    }
 
-  // Adding margin to rows
-  @include breakpoints.breakpoint-up(MinDesktop) {
-    $grid-gutter-width-ext: 56px;
-    $grid-margin-ext: 120px;
-
-    $grid-gutter-width-int: 24px;
-    $grid-margin-left-int: 88px;
-    $grid-margin-right-int: 32px;
-    .row.e-grid-gutters-ext {
-      margin: 0 calc((#{$grid-gutter-width-ext} / 2) * -1) !important;
-      [class^='col'] {
-        padding-right: calc((#{$grid-gutter-width-ext} / 2)) !important;
-        padding-left: calc((#{$grid-gutter-width-ext} / 2)) !important;
-      }
-      &.e-grid-gutters-vertical [class^='col'] {
-        padding-bottom: calc((#{$grid-gutter-width-ext} / 2)) !important;
-        padding-top: calc((#{$grid-gutter-width-ext} / 2)) !important;
-      }
+    &.e-grid-margin-ext {
```

Comment:
Are we the only ones using this? (https://github.com/search?q=org%3A3lvia%20e-grid-margin-ext&type=code) If neither the designers nor the developers use grids, then, in my opinion, we should delete it

Reply:
Totally agree! I also talked with Henrik to remove the use of `e-grid-margin-int` from Elvis, so that we can remove it as well.